credoai/metrics/metric_constants.py

```diff
@@ -62,3 +62,19 @@
     "demographic_parity_ratio",
     "demographic_parity_difference"
 ]
+
+# REGRESSION METRICS
+REGRESSION_FUNCTIONS = {
```

Comment:
Can you add to `METRIC_EQUIVALENTS` above? For instance, we probably want to add things like:
```
'mean_absolute_error': ['MAE'],
'mean_squared_error': ['MSE'],
...
```

Reply:
Added

```
    'mean_absolute_error': ['MAE'],
    'mean_squared_error': ['MSE'],
    'root_mean_squared_error': ['RMSE'],
    'r2_score': ['r_squared']
```